src/views/Home/components/PrizeTracks/PrizeTracks.scss

```diff
@@ -0,0 +1,107 @@
+@use "~stylesheets/typography";
+
+.prizeTracks {
+  background: linear-gradient(#b0b6f7, #b0b6f7);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  &__title {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    &__container {
+      width: 524px;
+      max-width: 90%;
+      height: 106px;
+      background-color: #32578E;
+      border-radius: 28px;
+      display: flex;
+      justify-content: center;
+      align-items: center; 
+      &--header {
+        @include typography.title;
+        font-size: 70px;
+        text-shadow: 5px 0px #777CE7;
+      }
+    }
+  }
+  
+  &__images {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    max-width: 1600px;
+    align-items: center;
+    &__container {
+      margin: 40px 50px;
+      width: 341px;
+      min-width: 260px;
+      aspect-ratio: 1/1;
```

Comment:
It's still not display on mobile correctly but I think it's because of aspect ratio not having a lot of browser support.